apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -317,6 +366,7 @@ export default connect(
               ...this.getDebugOutputBackgroundStyle()
             }}
           >
+            <div>{this.state.mercator}</div>
```

Comment:
Should we put this behind a flag while it's in development?

Reply:
Yup sounds good to me. I'll refactor it that way!